.changeset/tasty-seals-fail.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
🙃 I think we should add changelog since this PR is a fix for ticket flows for a bug introduced in the previous version.

Reply:
Not aware of the bug you're talking about. Do we have any resource on that. I'd like to make sure this fixes it as well? I'm just doing this because the state will be lost on refresh and that seems wrong.

Reply:
initiate a ticket flow and before completing it (eg before completing the sign-up from an invitation), refresh the page. Since the ticket query param is removed, the flow will never be completed.

Reply:
Seems to be no issue with multi-step flows as the ticket is passed through the `create()` method only. It's just the refresh from what I can tell which should be fixed by this PR.

Reply:
probably yes. i haven't checked all the flows. If that is the only flow affected, let's proceed with fixing only that.